sql/2025/07/row-block-allow-to-wrap-to-multiple-lines-usage.sql

```diff
@@ -0,0 +1,77 @@
+# HTTP Archive query to get % WordPress sites using Allow to wrap to multiple lines option in Row block.
+#
+# WPP Research, Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# See query results here: https://github.com/GoogleChromeLabs/wpp-research/pull/194
+
+DECLARE
+  DATE_TO_QUERY DATE DEFAULT '2025-07-01';
```

Comment:
I don't think this one is available yet, but we can test with the previous one for now.